.travis.yml

```diff
@@ -8,19 +8,20 @@ go:
 services:
   - docker
 
-go_import_path: gopkg.in/bblfsh/sdk.v2
+go_import_path: github.com/bblfsh/sdk
 
 matrix:
   fast_finish: true
   allow_failures:
     - go: tip
 
+env:
+  - GO111MODULE=on
 
 before_install:
-  - cd $GOPATH/src/gopkg.in/bblfsh/sdk.v2
+  - cd $GOPATH/src/github.com/bblfsh/sdk
```

Comment:
not sure why is this still needed?

Reply:
In theory, it shouldn't matter anymore. But I would prefer to first make it work like this, and then start cutting up unneeded things. There may be some other scripts that rely on this behavior, for example.